src/expr/src/agg/def.rs

```diff
@@ -237,6 +237,9 @@ pub enum AggKind {
     PercentileDisc,
     Mode,
     Grouping,
+
+    /// Return arbitrary one of the input values, and never change even on Update/Delete.
+    InternalArbitraryValue,
```

Comment:
Postgres 16 introduced a new function:
```
any_value ( anyelement ) → same as input type
  Returns an arbitrary value from the non-null input values.
```
https://www.postgresql.org/docs/16/functions-aggregate.html

Maybe it can be exposed as this one?

Reply:
I guess it's a little bit different, in that we need to output even NULL value. And, we assume all values are the same within one group, so there's no need to maintain a dedicated state table.